pkg/plugins/runtime/k8s/metadata/annotations.go

```diff
@@ -22,6 +22,11 @@ const (
 	// inbound listeners won't be generated in that case
 	KumaGatewayAnnotation = "kuma.io/gateway"
 
+	// KumaBuiltinGatewayAnnotation marks a pod as Kuma Gateway dataplane
+	// element. This annotation should only ever be applied by a Kuma control
+	// plane. It alters the dataplane generated by the pod controller.
+	KumaBuiltinGatewayAnnotation = "gateways.kuma.io/gateway-name"
```

Comment:
I'm thinking about how this would work in practice. For ZoneIngress, we have a kumactl command that generates deployment YAML, so I expect that we would so something similar in a follow-up PR. When we deploy dataplanes this way, we haven't bound them to a specific gateway yet, so is `gateways.kuma.io/gateway-name` the annotation we should be looking for?

Reply:
This is not used any more, so drop it from this PR?

Reply:
dropped